internal/chain/reward.go

```diff
@@ -294,9 +294,12 @@ func AccumulateRewardsAndCountSigs(
 					return network.EmptyPayout, err
 				}
 
-				shardExternalShare := shard.Schedule.InstanceForEpoch(
-					epoch,
-				).ExternalVotePercent()
+				shardExternalShare := numeric.ZeroDec()
```

Comment:
rename to allSignersShare.  Also do the same for beacon chain rewarding "beaconExternalShare"